src/main/java/org/datadog/jenkins/plugins/datadog/DatadogPeriodicCounterSender.java

```diff
@@ -0,0 +1,45 @@
+package org.datadog.jenkins.plugins.datadog;
+
+import hudson.Extension;
+import hudson.model.*;
+
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.concurrent.ConcurrentMap;
+import java.util.logging.Logger;
+import org.datadog.jenkins.plugins.datadog.model.ConcurrentMetricCounters;
+import org.datadog.jenkins.plugins.datadog.model.CounterMetric;
+
+@Extension
+public class DatadogPeriodicCounterSender extends PeriodicWork {
+
+    private static final Logger logger = Logger.getLogger(DatadogPeriodicCounterSender.class.getName());
+
+    @Override
+    public long getRecurrencePeriod() {
+        // run frequency - 15 seconds
+        return PeriodicWork.MIN * 60 / 4;
```

Comment:
This should be `return PeriodicWork.MIN / 4;` (every 15 seconds, not 15 minutes). Will update.